server/models/schema.sql

```diff
@@ -612,24 +612,21 @@ CREATE TABLE `employee` (
   `email`         VARCHAR(70) DEFAULT NULL,
   `fonction_id`   TINYINT(3) UNSIGNED DEFAULT NULL,
   `service_id`    SMALLINT(5) UNSIGNED DEFAULT NULL,
-  `location_id`   BINARY(16) NOT NULL,
   `creditor_uuid` BINARY(16) DEFAULT NULL,
-  `debtor_uuid`   BINARY(16) DEFAULT NULL,
   `locked`        TINYINT(1) DEFAULT NULL,
+  `patient_uuid`  BINARY(16) DEFAULT NULL,
   PRIMARY KEY (`id`),
   UNIQUE KEY `employee_1` (`code`),
   KEY `fonction_id` (`fonction_id`),
   KEY `service_id` (`service_id`),
-  KEY `location_id` (`location_id`),
   KEY `creditor_uuid` (`creditor_uuid`),
-  KEY `debtor_uuid` (`debtor_uuid`),
```

Comment:
For the hospital, we have creditors and debtors. 
- Creditors are providers of services or drugs in this we have (employees and suppliers)
- Debtors consumed services and drugs in this group we have (patients and other clients like visitors of guest house for the case of IMCK).
If you want we will discuss it, but I think it makes more sense to remove the ```debtor_uuid``` reference form ```employee``` table. As the **employee** must have a correspondent **patient** that why there is a ```patient_uuid``` reference in the employee table.